src/handler/reaction.jl

```diff
@@ -19,11 +19,17 @@ const NO_REACTION = Char[]
 # Reactors
 # ----------------------------------------------------------------------
 
+function contains_any(s::AbstractString, words::AbstractVector{String})
+    return any(occursin.(lowercase.(words), s))
```

Comment:
I think lowercase needs to be applied to s not words.
The line should be
`return any(occursin.(words,lowercase(s)))`